gamestonk_terminal/helper_funcs.py

```diff
@@ -26,7 +26,11 @@
 
 
 def check_valid_path(path: str) -> str:
-    if not os.path.exists(path):
+    if not os.path.exists(
+        os.path.abspath(
+            "gamestonk_terminal/portfolio_analysis/portfolios/" + path + ".csv"
```

Comment:
This path is not OS-agnostic and will probably fail on windows because of the different slashes.

Better change to something like

```python
os.path.abspath(
    os.path.join("gamestonk_terminal", "portfolio_analysis",  "portfolios", f"{path}.csv")
)
```

Reply:
Should be done now :)